.github/workflows/gating.yaml

```diff
@@ -84,3 +84,17 @@ jobs:
         pip install tox
     - name: Test '${{ matrix.tox_env }}' with tox
       run: tox -e ${{ matrix.tox_env }}
+
+  hadolint:
+    name: Hadolint
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
```

Comment:
If https://github.com/containerbuildsystem/cachi2/pull/5 gets merged first, we might want to update to actions/checkout@v3

Reply:
I'd go with v3 even before merging #5